v2/robotmk/api.py

```diff
@@ -0,0 +1,34 @@
+from pydantic import BaseModel
+
+from robotmk import parse_xml
+
+
+class Test(BaseModel, frozen=True):
+    name: str
+    id_: str
+
+
+class Result(BaseModel, frozen=True):
+    suite_name: str
+    tests: list[Test]
+    xml: str
+
+
+def create(suite_name: str, xml: str) -> Result:
```

Comment:
`create_result` maybe? Otherwise, this could also become a classmethod of `Result`.

Reply:
I did not make this a `classmethod`, because `create` depends on `parse_xml`. `parse_xml` does not belong into the `robotmk_api` and thus `create` cannot be part of the `robotmk_api`. In fact, that function does not belong in that module, but into it's own module. However, until we actually start addressing the API problem, these considerations should not really affect our product, so I didn't create the extra module.

`create_result` will probably have to be changed as well, when we add a second JSON model, but for now this is ok with me.

Reply:
Ok.